lib/mongo_client.js

```diff
@@ -206,6 +207,32 @@ class RemoteMongoDBCollection {
             upsert: options.upsert,
         });
     }
+
+    async* watch({ids = undefined, filter = undefined} = {}) {
+        let args = {
+            database: this.databaseName,
+            collection: this.collectionName,
+            ids,
+            filter,
+        };
+
+        const request = this.user._makeStreamingRequest("watch", this.serviceName, this.user._cleanArgs([args]));
+        const reply = await (new DefaultNetworkTransport().fetch(request));
```

Comment:
The network transport is ment to be reused throughout the lifetime of the runtime. Do you see a way to avoid instantiating it upon every call to this function?

Reply:
I could, but isn't it really cheap to construct? It looks like after the first call of the constructor, all other constructor calls don't really do anything.